### PR TITLE
Fix daily meeting

### PR DESCRIPTION
**Pull request by sonntuet1997**

### CODE REVIEW

**Review by Draply — APPROVED** [review]
LGTM!

**Review by ktokto313 — DISMISSED** [review]

**Comment by ktokto313** [comment]
I didn't expect to see code related to obsidian here but it's fine.

**Comment by sonntuet1997** [comment]
> I didn't expect to see code related to obsidian here but it's fine by me.

I commit this so when you pull the code, Obsidian will read it and apply the config within it so you don't have to do it manually.

File: README.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ This repository serves as a centralized hub for our agreed-upon policies, embody
...
 
 ## Policy Categories
 
-- **General Policies:** Refer to [Policies.md](policies.md) for overarching guidelines that apply to various aspects of our operations.
+- **General Policies:** Refer to [Policies](policies.md) for overarching guidelines that apply to various aspects of our operations.
 
-- **Planning:** Explore [Planning.md](planning.md) for policies related to the strategic and organizational planning processes.
+- **Planning:** Explore [Planning](planning.md) for policies related to the strategic and organizational planning processes.
 
 - **Meetings:** Find guidelines for meeting conduct and participation in [Meetings.md](meetings.md).
```
Comment (haipro287) [inline README.md:15]: It should be Meetings without.md?

**Review by haipro287 — CHANGES_REQUESTED** [review]

**Review by haipro287 — APPROVED** [review]

**Review by VanhGer — APPROVED** [review]

**Review by ktokto313 — APPROVED** [review]

**Pull request merged**